server/admin/types.go

```diff
@@ -42,10 +43,20 @@ func (at *APITime) MarshalJSON() ([]byte, error) {
 // UnmarshalJSON unmarshals JSON string containing a time in RFC3339 format with
 // millisecond precision into an APITime.
 func (at *APITime) UnmarshalJSON(b []byte) error {
-	t, err := time.Parse(RFC3339Milli, string(b))
+	if len(b) < 2 {
+		return nil
+	}
+	// Parenthesis are included in b and must be removed.
+	t, err := time.Parse(RFC3339Milli, string(b[1:len(b)-1]))
 	if err != nil {
 		return nil
 	}
 	at.Time = t
 	return nil
 }
```

Comment:
I believe this was creating a bug where the time always unmarshalled to nil, or zero time.